server/routes/auth.js

```diff
@@ -0,0 +1,13 @@
+import Router from 'express';
+import AuthController from '../controllers/authController';
+import bnLog from '../helpers/log.util';
+import validateAuth from '../middlewares/validateAuth';
+// eslint-disable-next-line no-unused-vars
```

Comment:
Resolve this issue as well. You should it disable eslint rules. Refer to their documentation on how to fix its issues.